src/misc/8.md

```diff
@@ -0,0 +1,42 @@
+# Misc 8 @adotinthevoid
+
+{{#include ../include/quiz-is-wip.md}}
+
+```rust
+{{#include ../../code/examples/misc_8.rs:matters}}
+```
+
+<details>
+<summary>Solution</summary>
+
+```
+{{#include ../../code/examples/stderr/misc_8.stderr}}
+```
+
+There are 3 kinds of structs in rust:
+
+1. Plain structs (eg `struct Foo{bar: i32}`)
+2. Tuple structs (eg `struct Bar(i32);`)
+3. Unit structs (eg `struct Baz;`)
+
+However all 3 kinds can have a struct with no fields.
```

Comment:
```suggestion
All 3 kinds can have a struct with no fields.
```
I'd drop the "however" here, I don't think it fits very well (maybe remove the entire sentence?)